src/app/shared/components/carousel/carousel.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
```

Comment:
Seems to be a useless component just to add a few classes to a swiper element.
What is the purpose?

Reply:
My thought was to create a reusable swiper-container that has these classes.